static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -128,6 +128,8 @@ declare type InitEpicABTest = {
     template?: EpicTemplate,
     deploymentRules?: DeploymentRules,
     hasCountryName?: boolean,
+    minArticleViews?: ?number,
```

Comment:
double `?` here because I want a value of `undefined` to be possible, but I also want setting the field to be optional. What a mess